src/sentry/integrations/slack/endpoints/command.py

```diff
@@ -38,6 +45,57 @@ def send_ephemeral_notification(self, message: str) -> Response:
             }
         )
 
+    def link_user(self, slack_request: SlackCommandRequest) -> Response:
+        if slack_request.has_identity:
+            return self.send_ephemeral_notification(
+                ALREADY_LINKED_MESSAGE.format(username=slack_request.identity_str)
+            )
+
+        integration = slack_request.integration
+        organization = integration.organizations.all()[0]
+        associate_url = build_linking_url(
+            integration=integration,
+            organization=organization,
+            slack_id=slack_request.user_id,
+            channel_id=slack_request.channel_id,
+            response_url=slack_request.response_url,
+        )
+        return self.send_ephemeral_notification(
+            LINK_IDENTITY_MESSAGE.format(associate_url=associate_url)
```

Comment:
Reusing this text feels weird to me, because as a user I'm typing `/sentry link` and then it says "Looks like you haven't linked your Sentry account with your Slack identity yet! <{associate_url}|Link your identity now> to perform actions in Sentry through Slack." which is like "yeah, I know, that's why I typed it". Maybe instead you could make a new string say something like: "Link your Slack identity to your Sentry account to receive notifications. You'll also be able to perform actions in Sentry through Slack." I think we'd still want to keep the old string as it makes sense in the context of trying to click on an action button and not being able to do it until you link.

Reply:
I updated the copy.